### ISSUE TITLE
Encoder/decoder for 'oid'

### ISSUE THREAD

**Issue by int-index** [nikita-volkov/hasql#106]
I want to query PostgreSQL system tables, such as `pg_type`:

![image](https://user-images.githubusercontent.com/4061728/51475021-3699f180-1d92-11e9-8597-a8834d70cc76.png)

There are stock decoders for `int2`, `bool`, `char`, etc, but I'm not sure what to use for columns of type `oid` or `name`.

**Comment by nikita-volkov** [comment]
I agree that it should be added. If you wish to make a PR, please start with the "postgresql-binary" library, implementing a test for the codec in the same style as the other tests there.

**Comment by nikita-volkov** [comment]
Implemented in:

- https://hackage.haskell.org/package/hasql-1.10.2.3/docs/Hasql-Encoders.html#v:oid
- https://hackage.haskell.org/package/postgresql-types-0.1.1.1/docs/PostgresqlTypes-Oid.html
